packages/um7/package.py

```diff
@@ -24,6 +24,7 @@ class Um7(Package):
 
     # https://code.metoffice.gov.uk/trac/um/wiki/PastReleases
     version("7.3")
+    version("access-esm1.5", branch="access-esm1.5")
```

Comment:
Do you want to use `preferred=True` like you did in the MOM5 SPD for one of the `version()` calls?

Reply:
I am not sure. I don't think that it is needed until we add "access-esm1.6". 
If I did, add `preferred=True`, it would be
```
version("7.3", branch="main", preferred=True)
```
and I don't think that would be useful until ACCESS-ESM1.6 changes are merged into `main` as per https://github.com/ACCESS-NRI/UM_v7/issues/8

What do you think?

Reply:
Since you found `preferred=True`, my preference is to have one version set as preferred when there are multiple versions and some are non-numeric.

Reply:
Will do.

Reply:
Done.